src/openms/include/OpenMS/ANALYSIS/OPENSWATH/OpenSwathOSWWriter.h

```diff
@@ -108,10 +112,16 @@ namespace OpenMS
 
       // Create SQL structure
       const char * create_sql =
+        "CREATE TABLE RUN(" \
+        "ID INT PRIMARY KEY NOT NULL," \
+        "FILENAME TEXT NOT NULL); " \
```

Comment:
maybe we could have another field here, like a comment but (i) I am not sure we need it and (ii) we can always add it later